easybuild/easyconfigs/c/CDO/CDO-1.9.8-foss-2018b.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'CDO'
+version = '1.9.8'
+
+homepage = 'https://code.zmaw.de/projects/cdo'
+description = """CDO is a collection of command line Operators to manipulate and analyse Climate and NWP model Data."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+# stick to lowopt (-O1) to avoid internal compiler error when building on Intel Skylake
+toolchainopts = {'pic': True, 'usempi': True, 'lowopt': True}
+
+source_urls = ['https://code.mpimet.mpg.de/attachments/download/20826/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['f2660ac6f8bf3fa071cf2a3a196b3ec75ad007deb3a782455e80f28680c5252a']
+
+dependencies = [
+    ('HDF5', '1.10.2'),
+    ('netCDF', '4.6.1'),
+    ('YAXT', '0.6.2'),
+    ('ecCodes', '2.17.0', '-Python-3.6.6'),
```

Comment:
@tstrempel If you depend on something that depends on Python, this should be reflected in the `versionsuffix` (this is mostly done to make it clear which modules depend on Python 2 vs 3, to avoid clashes).

```python
name = 'CDO'
version = '1.9.8'
versionsuffix = '-Python-3.6.6'

...

dependencies = [
    ...
    ('ecCodes', '2.17.0', versionsuffix),